### ISSUE TITLE
Evaluate JScholarship metadata - reducing the number of metadata fields 

### ISSUE THREAD

**Issue by htpvu** [jhu-idc/iDC-general#21]
**Tasks that should be done beforehand:** None
**External dependencies:** Katie (confirming selection of metadata fields for the 5 initial collections) 

- Evaluate and gather statistics on current JScholarship fields: overall usage, usage by community, last time created
- Select fields to be retired
- Determine what values from retired fields will be mapped to other fields and what values can be deleted
- Map to-be-retired fields & values to remaining fields
-- Create mapping in spreadsheet
-- Determine what clean-up should be done to merge into new fields
----- Mapping to controlled vocabularies?
----- Standardizing to particular data formatting
----- Note?
- Confirm selection of any metadata fields & values to be deleted with Katie

additional context https://docs.google.com/document/d/1C3cA5TZLKcYsKEThITAfxS1Nc6AE8PVUeclZhNczisM/edit#

**Comment by mjanowiecki** [comment]
Hi! The information for this issue can be found in the following folder: https://drive.google.com/drive/folders/1I0Ou1JueWLGBQJYjX2VgI50XD6LTIKoi?usp=sharing 

I ended up not making as many concrete recommendations about merging/deleting specific elements as I initially thought, but I think these documents will give us a good baseline for discussing these issues in the metadata committee and making formal decisions then.
